packages/core/src/Modules/Endpoint/Endpoint.jsx

```diff
@@ -1,24 +1,38 @@
 import React from 'react';
 import { Field, Form, Formik } from 'formik';
 import { Button, Input, Checkbox, Text } from '@deriv/components';
-import { getDebugServiceWorker, getAppId, getSocketURL, PlatformContext, isMobile } from '@deriv/shared';
+import {
+    getDebugServiceWorker,
+    getAppId,
+    getSocketURL,
+    PlatformContext,
+    isMobile,
+    TRADE_FEATURE_PREFIX,
+    website_domain,
+} from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 
 const FeatureFlagsSection = observer(() => {
     const { feature_flags } = useStore();
-
+    const visible_feature_flags = Object.entries(feature_flags.data ?? {})?.reduce(
+        (a, [key, value]) => ({
+            ...a,
+            ...(location.hostname === website_domain && key.startsWith(TRADE_FEATURE_PREFIX) ? {} : { [key]: value }),
+        }),
+        {} // hiding flags for trade types in production
+    );
```

Comment:
Yeah I believe the changes in this file are not needed!

Reply:
@farzin-deriv @matin-deriv this is what I discussed with you earlier: the flags for trade features only will be hidden for app.deriv.com because our DTrader API will not provide any flags like it's done for wallets, it means we don't want our users in production to check/uncheck trade feature flags. It won't affect any other flags that don't start with 'trade_' :)

Reply:
I think I've found a better approach to this instead of using prefix.